webv2/web.go

```diff
@@ -36,6 +36,7 @@ import (
 	"time"
 
 	instance "cloud.google.com/go/spanner/admin/instance/apiv1"
+	storageaccessor "github.com/GoogleCloudPlatform/spanner-migration-tool/accessors/storage"
```

Comment:
can we make this camel case? Same comment for other locations as well.

Reply:
Replied [here](https://github.com/Deep1998/harbourbridge/pull/12/files#r1462823339)